osv/models.py

```diff
@@ -722,8 +722,8 @@ def to_vulnerability(self, include_source=False, include_alias=True):
     aliases = []
 
     if include_alias:
-      related_bugs = Bug.query(Bug.related == self.db_id).fetch()
-      related_bug_ids = [bug.db_id for bug in related_bugs]
+      related_bugs = Bug.query(Bug.related == self.db_id).fetch(keys_only=True)
+      related_bug_ids = [bug.id() for bug in related_bugs]
```

Comment:
hmm, is this going to work? `bug.id()` may not be the same as `bug.db_id` for historical reasons (though we can probably clean that up via a database migration)

Reply:
ref: https://github.com/google/osv.dev/issues/2786